sfge/src/test/java/com/salesforce/graph/source/supplier/ExposedControllerMethodSupplierTest.java

```diff
@@ -0,0 +1,64 @@
+package com.salesforce.graph.source.supplier;
+
+import com.salesforce.collections.CollectionUtil;
+import com.salesforce.metainfo.MetaInfoCollectorTestProvider;
+import com.salesforce.metainfo.VisualForceHandlerImpl;
+import java.util.List;
+import java.util.TreeSet;
+import org.junit.jupiter.api.Test;
+
+public class ExposedControllerMethodSupplierTest extends BaseSourceSupplierTest {
+
+    /** Public/global methods on known controllers should be covered by this supplier. */
+    @Test
+    public void supplierLoadsControllerMethods() {
```

Comment:
Equivalent to `PathEntryPointUtilTest#getPathEntryPoints_includesExposedcontrollerMethods` and `PathEntryPointUtilTest#testGetExposedControllerMethods`.